docker-compose/preclinical/Dockerfile

```diff
@@ -1,12 +1,22 @@
-FROM openjdk:8-jdk-alpine
-RUN apk add bash
```

Comment:
Installing bash was quite usefull to be able to connect to the server. Is it installed by default with stretch ?

Reply:
Hi Jean-Come, I have no idea, and as it is not my expertise, could you change it as you like it?

Reply:
I think bash is installed by default on stretch (it was not on alpine) so it's OK.
Jean-Côme